src/neural/network_demux.cc

```diff
@@ -162,7 +163,8 @@ class DemuxingNetwork : public Network {
     }
   }
 
-  void Worker() {
+  void Worker(int id) {
+    Numa::BindThread(id);
```

Comment:
Maybe this would be better as `id+1` to leave space for an active search thread on the first group?

Reply:
Sounds logical to me.